lib/web_ids.js

```diff
@@ -62,18 +73,41 @@ function performChecks(message) {
   checkRequestUri(message.body);
 }
 
+/**
+ * Check if ip address corresponds to one of the ips in signature database
+ * @param {string} srcIp ip address in the request message
+ * */
 function checkIp(srcIp) {
   // TODO
   // file with signatures should be read and ip's stored
   // match them here
 }
 
+/**
+ * Check if request body contains offending data
+ * @param {Object} requestBody JSON containing request body data
+ * */
 function checkRequestBody(requestBody) {
   // TODO
   // file with signatures should be read and request uri's stored
   // match them here
 }
-function checkRequestUri(requestUri) {
```

Comment:
why deleted?

Reply:
This function is replaced with checkRequestHeader, since that better reflects signatures JSON files.

Reply:
I think it is better to rename the signature to uriSignature since head is a bit missleading

Reply:
I think you're right, @geo-bert thoughts?

Reply:
uriSignature would be wrong because it checks the entire header and not just the uri. I don't find checkRequestHeader misleading.

Reply:
but every information we get from the logs except the body is part of the header. And the other signature names have t method or uri and are only checking the status for instance.
For me, the name tells me what the main parameter is in a signature. the other ones are for additional info

Reply:
Maybe checkMethodAndUri would be better

Reply:
> 
> 
> but every information we get from the logs except the body is part of the header. And the other signature names have t method or uri and are only checking the status for instance.
> For me, the name tells me what the main parameter is in a signature. the other ones are for additional info

You are absolutely right. I now see why this would be misleading. This should then also be changed in the signature file.

Reply:
I will change the `headerSignature` in JSON files to `uriSignature`. Request Method is right now included in almost every signature. I would rather open an issue and discuss `requestMethod` there.